drivers/npmx/npmx_shell.c

```diff
@@ -3578,132 +3585,109 @@ static int cmd_timer_config_prescaler_get(const struct shell *shell, size_t argc
-
 	return 0;
 }
 
-static int cmd_vbusin_current_limit_get(const struct shell *shell, size_t argc, char **argv)
+static int cmd_vbusin_status_connected_get(const struct shell *shell, size_t argc, char **argv)
 {
 	ARG_UNUSED(argc);
 	ARG_UNUSED(argv);
 
-	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
-
-	if (npmx_instance == NULL) {
-		shell_error(shell, "Error: shell is not initialized.");
+	npmx_vbusin_t *vbusin_instance = vbusin_instance_get(shell);
+	if (vbusin_instance == NULL) {
 		return 0;
 	}
 
-	npmx_vbusin_current_t current_limit;
-	npmx_vbusin_t *vbusin_instance = npmx_vbusin_get(npmx_instance, 0);
-	npmx_error_t err_code = npmx_vbusin_current_limit_get(vbusin_instance, &current_limit);
-
+	uint8_t status_mask;
+	npmx_error_t err_code = npmx_vbusin_vbus_status_get(vbusin_instance, &status_mask);
 	if (!check_error_code(shell, err_code)) {
-		shell_error(shell, "Error: unable to read VBUS current limit.");
+		print_get_error(shell, "VBUS connected status");
 		return 0;
 	}
 
-	uint32_t current;
-
-	if (npmx_vbusin_current_convert_to_ma(current_limit, &current)) {
-		shell_print(shell, "Value: %u mA.", current);
-	} else {
-		shell_error(shell, "Error: unable to convert VBUS current limit value.");
-	}
-
+	print_value(shell, !!(status_mask & NPMX_VBUSIN_STATUS_CONNECTED_MASK), UNIT_TYPE_NONE);
 	return 0;
 }
 
 static int cmd_vbusin_current_limit_set(const struct shell *shell, size_t argc, char **argv)
 {
-	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
-
-	if (npmx_instance == NULL) {
-		shell_error(shell, "Error: shell is not initialized.");
+	args_info_t args_info = { .expected_args = 1,
+				  .arg = {
+					  [0] = { SHELL_ARG_TYPE_UINT32_VALUE, "current limit" },
+				  } };
+	if (!arguments_check(shell, argc, argv, &args_info)) {
 		return 0;
 	}
 
-	if (argc < 2) {
-		shell_error(shell, "Error: missing current limit value.");
+	npmx_vbusin_t *vbusin_instance = vbusin_instance_get(shell);
+	if (vbusin_instance == NULL) {
 		return 0;
 	}
 
-	int err = 0;
-	uint32_t current = shell_strtoul(argv[1], 0, &err);
-	npmx_vbusin_t *vbusin_instance = npmx_vbusin_get(npmx_instance, 0);
+	uint32_t current_limit_ma = args_info.arg[0].result.uvalue;
+	npmx_vbusin_current_t vbusin_current = npmx_vbusin_current_convert(current_limit_ma);
+	if (vbusin_current == NPMX_VBUSIN_CURRENT_INVALID) {
+		print_convert_error(shell, "milliamperes", "current limit");
+		return 0;
+	}
 
-	if (err != 0) {
-		shell_error(shell, "Error: current limit must be an integer.");
+	npmx_error_t err_code = npmx_vbusin_current_limit_set(vbusin_instance, vbusin_current);
+	if (!check_error_code(shell, err_code)) {
+		print_set_error(shell, "current limit");
 		return 0;
 	}
 
-	npmx_vbusin_current_t current_limit = npmx_vbusin_current_convert(current);
+	print_success(shell, current_limit_ma, UNIT_TYPE_MILLIAMPERE);
+	return 0;
+}
+
+static int cmd_vbusin_current_limit_get(const struct shell *shell, size_t argc, char **argv)
```

Comment:
`current_limit` should be first, alphabetical order. Same in command creation at the bottom of the file